.changeset/public-hands-relate.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ebay/ebayui-core": patch
```

Comment:
Double changeset? You should probably remove one of these.

Reply:
There are two changes in this, two issues